mmv1/templates/terraform/examples/network_security_security_profile_group_basic.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_network_security_security_profile_group" "<%= ctx[:primary_resource_id] %>" {
+  provider                  = google-beta
+  name                      = "<%= ctx[:vars]['resource_name'] %>"
+  parent                    = "organizations/<%= ctx[:test_env_vars]['org_id'] %>"
+  description               = "my description"
+  threat_prevention_profile = "my-security-profile"
```

Comment:
This is currently failing with the API error:

```
googleapi: Error 400: The request was invalid: SPG is invalid: Invalid value for resource name: my-security-profile. Must be a match of regex '^/?[a-z]*/[^/]*/locations/[^/]*/securityProfiles/[a-z]([-a-z0-9]{0,61}[a-z0-9])?$': invalid argument
```

Generally for an example like this we'd include the parent resource in the example (generating its ID as a context variable) to show how they can be used together.

Reply:
Yep! I was waiting for security_profile ([PR here](https://github.com/GoogleCloudPlatform/magic-modules/pull/9800)) to be merged. Now that it's available, I'll fix it.

Reply:
done